.gitignore

```diff
@@ -41,3 +41,5 @@ venv/
 ENV/
 env.bak/
 venv.bak/
+.secrets
+secrets
```

Comment:
@thompsgj Can we have a line break here?

Reply:
@raouldevil   Sounds good, thanks for that suggestion.  I've added a label to the secrets lines since they are different from the preceding entries.  I've added the line break, as well.